wiki/Mapping_Techniques/Overlap/en.md

```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Overlap
+
+![Rotational_Symmetry.png](/img/overlap.jpg "An example of a hitcircle overlapping a sliderhead.")
```

Comment:
```suggestion
![Rotational_Symmetry.png](img/overlap.jpg "An example of a hitcircle overlapping a sliderhead.")
```

the beginning `/` in the link basically means to start with the root folder, to use current directory remove it or use `./`